docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -36,9 +36,241 @@ You get ready-to-use code tailored to your needs!
 
 ### Setting up the project
 
-This guide assumes you have a React project. You can create a basic React project using [Vite](https://vitejs.dev/). Refer to the [React documentation](https://react.dev/learn/start-a-new-react-project) to learn how to set up a project in the framework.
+This guide assumes you have a ready React project. If not, you can create a basic React project using [Vite](https://vitejs.dev/). Refer to the [React documentation](https://react.dev/learn/start-a-new-react-project) to learn how to set up a project in the framework.
 
-## Component properties
+### Installing from npm
```

Comment:
@Mati365 is it valid?

Reply:
@illia-stv Tricky question. React integration is installed from NPM, but it can use `useCKEditorCloud` which uses CDN under the hood. I'd keep `NPM` because we describe `@ckeditor/ckeditor5-react` here. imho

Reply:
I would agree with that, lets stay with `NPM` then.

Reply:
Let's make it more specific, like Install React component package or something like that

Reply:
I'm fine with npm, then.